nexus/db-queries/src/db/datastore/silo_group.rs

```diff
@@ -199,6 +199,8 @@ impl DataStore {
 
         let group_id = authz_silo_group.id();
 
+        // Prefer to use "transaction_retry_wrapper"
```

Comment:
Add this one to the list of those to be converted?

Reply:
Added to https://github.com/oxidecomputer/omicron/issues/7213